model/network.py

```diff
@@ -1,44 +1,24 @@
-from mongoengine import *
-from cassandra.cqlengine.models import Model
-from cassandra.cqlengine import columns
+from utils.database import Base
+from sqlalchemy import Column, String, Boolean, ForeignKey
 
-class Network(Model):
-    cidr = columns.Text(max_length=50)
-    project = columns.Text(primary_key=True, max_length=50)
-    nw_name = columns.Text(primary_key=True, max_length=50)
-    created = columns.Boolean(required=True, default=False)
 
+class Network(Base):
+    
+    __tablename__ = 'network'
+    
+    project = Column(String, ForeignKey("project.name", onupdate='CASCADE', ondelete='CASCADE'), primary_key=True, unique=True)
+    nw_name = Column(String, primary_key=True, unique=True)
+    cidr = Column(String, primary_key=True)
+    created = Column(Boolean, nullable=False, default=False)
 
-class NetworkMongo(Document):
-    cidr = StringField(required=True, max_length=50)
-    project = StringField(required=True, max_length=50)
-    nw_name = StringField(required=True, max_length=50)
-    created = BooleanField(required=True, default=False)
-    meta = {
-        'indexes': [
-            {'fields': ('cidr', 'project','nw_name'), 'unique': True}
-        ]
-    }
 
-
-class Subnet(Model):
-    cidr = columns.Text(primary_key=True, max_length=50)
-    nw_name = columns.Text(required=False, max_length=100)
-    project = columns.Text(primary_key=True, max_length=50)
-    subnet_name = columns.Text(primary_key=True, max_length=150)
-    subnet_type = columns.Boolean(required=True)
-    created = columns.Boolean(required=True, default=False)
-
-
-class SubnetMongo(Document):
-    cidr = StringField(required=True, max_length=50)
-    nw_name = StringField(required=False, max_length=100)
-    project = StringField(required=True, max_length=50)
-    subnet_name = StringField(required=True, max_length=150)
-    subnet_type = BooleanField(required=True)
-    created = BooleanField(required=True, default=False)
-    meta = {
-        'indexes': [
-            {'fields': ('cidr', 'project','subnet_name'), 'unique': True}
-        ]
-    }
\ No newline at end of file
+class Subnet(Base):
+    
+    __tablename__ = 'subnet'
+    
+    project = Column(String, ForeignKey("project.name", onupdate='CASCADE', ondelete='CASCADE'), primary_key=True, unique=True)
```

Comment:
there should be foreign key primary key relation between network table and subnet table

Reply:
Network name is already a foreign key in Subnet.